cluster/manifests/appdynamics/daemonset.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    name: monitor
+    application: appdynamics-machineagent
+    version: "4.3-0.74"
+  name: appdynamics-machineagent
+  namespace: kube-system
```

Comment:
As per our understanding daemonsets are supposed to run in kube-system. Is that correct?